protocol/migrations/sqlite/1676315400_add_deleted_column_to_activity_center_notifications_table.up.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE activity_center_notifications
+  ADD COLUMN deleted BOOLEAN NOT NULL DEFAULT FALSE;
+
+CREATE INDEX activity_center_chat_id_deleted_dismissed_accepted_idx
```

Comment:
These are the new indexes @cammellos. I've checked a couple queries in the `activity_center_persistence.go` file, and SQLite reported they were using indexes as expected. Is this good to go in your opinion? Anything you wish could be better?